modules/tinymce/src/plugins/fullscreen/main/ts/core/Keyboard.ts

```diff
@@ -0,0 +1,21 @@
+import { Cell } from '@ephox/katamari';
+
+import Editor from 'tinymce/core/api/Editor';
+import VK from 'tinymce/core/api/util/VK';
+
+import { FullScreenInfo } from './Actions';
+
+const setup = (editor: Editor, fullscreenState: Cell<FullScreenInfo | null>): void => {
+  editor.on('init', () => {
+    editor.on('keydown', (e) => {
+      if ((e.keyCode === VK.TAB || (e.keyCode === VK.TAB && e.shiftKey)) && fullscreenState.get()) {
```

Comment:
Your `|| (e.keyCode === VK.TAB && e.shiftKey)` did nothing, because it would only run if `e.keyCode === VK.TAB` was false.

Instead, what you probably want is to only handle tab with no modifiers other than shift.

```suggestion
      if (e.keyCode === VK.TAB && !(e.metaKey || e.ctrlKey) && fullscreenState.get()) {
```